3_javascript/javascript2/week3/Exercise1-2/script.js

```diff
@@ -0,0 +1,128 @@
+//1.2 functions - Log out the text Called after 2.5 seconds
+function delayedFunction(delay, stringToLog) {
+  setTimeout(function () {
+    console.log(stringToLog);
+  }, delay * 2500);
+}
```

Comment:
Why did you multiply by 2500? The `setTimeout` function expects the delay in milliseconds. If your function takes seconds, you should multiply it by 1000 to correctly convert it.